pyaedt/modeler/PrimitivesCircuit.py

```diff
@@ -10,6 +10,14 @@ class CircuitComponents(object):
     """CircutComponents class.
 
     This is the common class for managing all circuit components for Nexxim and Simplorer.
```

Comment:
```suggestion
    Manages all circuit components for Nexxim and Simplorer.
```

Reply:
I changed to fit the format for descriptions in the summary table listing all classes.